.github/workflows/ci.yml

```diff
@@ -59,10 +59,10 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: 3.9
-      - name: Install setuptools and wheel
-        run: python -m pip install --upgrade --user setuptools wheel
+      - name: Install build
+        run: python -m pip install --user build
       - name: Build sdist and wheel
-        run: python setup.py sdist bdist_wheel
+        run: python -m build
```

Comment:
_All_ of these lines can be replaced with:

```
- name: Build sdist and wheel
  run: pipx run build
```

GHA supports pipx without setup-python, and build supports pipx too.

Reply:
Interesting, thanks! I'd like to keep the scope of this PR restricted to removing setup.py, but feel free to open an issue.